_includes/docs/user-guide/ui/unit-conversion.md

```diff
@@ -0,0 +1,114 @@
+* TOC
+{:toc}
+
+{% assign sinceVersion = "4.1" %}
+{% include templates/since.md %}
+
+Starting from ThingsBoard 4.1, widgets support automatic data conversion and unit switching based on the selected unit system. 
+This is especially useful in global IoT solutions and significantly improves data interaction for users across different geographic regions where various measurement standards are used — such as the metric system (°C, km, kg) and the imperial system (°F, miles, lbs).
+
+<br><b><font size="4">Key benefits:</font></b>
+
+- Automatic conversion of numerical values into the appropriate units directly within the widget — no additional data processing required.
+- Flexible widget configuration that eliminates the need to duplicate widgets or create separate dashboards for users with different regional unit standards.
+- Support for a wide range of measurement units, allowing users to view values in the units that are most familiar and convenient to them.
+
+<b><font size="4">Where the unit conversion feature can be applied?</font></b>
+
+All widgets designed to display telemetry data include a unit conversion option.   
+Unit conversion is supported for telemetry data, Y-axis scales, and threshold settings.
+
+> **Note:** If a widget contains multiple telemetry keys, the unit conversion feature must be configured individually for each key.
+
+## Enable unit conversion
+
+Unit conversion for telemetry keys, the Y-axis, and threshold values is configured in the widget&#39;s "**Unit settings**" section.
+- Specify the **source unit** – This tells the system which unit the current value is measured in, so it knows how to convert it.
```

Comment:
This is not clear. Alternatives:
The source unit specifies the units of the incoming value from the back-end, enabling correct conversion.
The system uses the source unit to understand the units of the value received from the back-end and convert it appropriately.
To accurately convert the value sent from the back-end, the source unit must define its original units.